### PR TITLE
Adding test to validate target inside task costruct.

### PR DESCRIPTION
**Pull request by spophale**

### CODE REVIEW

File: tests/4.5/task/test_task_target.c [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+//
+////===----------------------------------------------------------------------===//
+
+#include <stdio.h>
+#include <omp.h>
+#include "ompvv.h"
+
+#define N 1000
+
+int test_task_target(){
```
Comment (jhdavis8) [inline tests/4.5/task/test_task_target.c]: Minor coding standard fix: space between close paren and open curly brace (also on lines 44 and 50)

File: tests/4.5/task/test_task_target.c [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  int a[N], b[N], c[N];
+  int errors=0, i;
```
Comment (jhdavis8) [inline tests/4.5/task/test_task_target.c]: Another coding standard thing: pad assignment operator with spaces

File: tests/4.5/task/test_task_target.c [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+#pragma omp task shared(c) private(i)
+  #pragma omp target map(from: c) map(to:a,b)
+    #pragma omp parallel for
+      for (i = 0; i < N; i++)
+        c[i] = a[i] + b[i];
+
+// #pragma omp taskwait
+
+  for (i = 0; i < N; i++){
+    OMPVV_TEST_AND_SET(errors, (c[i] != i+10));
```
Comment (jhdavis8) [inline tests/4.5/task/test_task_target.c]: Coding standard: pad addition operator with spaces

**Comment by jhdavis8** [comment]
Minor changes, otherwise looks good to me.

File: tests/4.5/task/test_task_target.c (hunk 1 above)
Comment (spophale) [reply tests/4.5/task/test_task_target.c]: Addressed.

File: tests/4.5/task/test_task_target.c (hunk 2 above)
Comment (spophale) [reply tests/4.5/task/test_task_target.c]: Addressed.

**Comment by jhdavis8** [comment]
Test is passing both compilers on fatnode.

**Pull request merged**